clamonacc/fanotif/fanotif.c

```diff
@@ -83,14 +83,14 @@ cl_error_t onas_setup_fanotif(struct onas_context **ctx)
     (*ctx)->fan_mask = fan_mask;
 
     if (optget((*ctx)->clamdopts, "OnAccessPrevention")->enabled && !optget((*ctx)->clamdopts, "OnAccessMountPath")->enabled) {
-        logg(LOGG_DEBUG, "ClamFanotif: kernel-level blocking feature enabled ... preventing malicious files access attempts\n");
-        (*ctx)->fan_mask |= FAN_ACCESS_PERM | FAN_OPEN_PERM;
+        logg(LOGG_DEBUG, "ClamFanotif: kernel-level blocking feature enabled ... preventing malicious file execution attempts\n");
```

Comment:
If we are scanning all file access, but only execution or modification are blocked with prevention, we may get a few confused users who see they are able to read files before the scan completes.  

I suggest putting some of this wording into the comments for `OnAccessPrevention` in both:
- `clamd.conf.sample`
- `clamd.conf.5.in`